kolibri/core/assets/src/views/AttemptLogList.vue

```diff
@@ -153,8 +153,8 @@
     displaySectionTitle,
     enhancedQuizManagementStrings,
   } from 'kolibri-common/strings/enhancedQuizManagementStrings';
+  import { coreStrings } from 'kolibri.coreVue.mixins.commonCoreStrings';
```

Comment:
Can't use `import coreStrings from 'kolibri.utils.coreStrings` from here

Reply:
This is outdated but I get errors on my end locally without changing the import here.